### PR TITLE
feature(login, join): show terms and conditions when logging in.

### PR DESCRIPTION
**Pull request by evans-g-crsj**

### CODE REVIEW

File: src/components/Login/roleForm.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.text-align-left {
+    text-align: left;
+}
+
+.section-header-spacing {
+    color: #2b388f;
```
Comment (francisl) [inline src/components/Login/roleForm.css:6]: why this is not part of the theme?

File: src/components/Login/termsConditions.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
+.footer {
+  text-align: right;
+  height: 52px;
+  border-top: 1px solid #f0f0f0;
```
Comment (francisl) [inline src/components/Login/termsConditions.css:4]: ☝️

File: src/App.js [hunk 3]
```diff
@@ -21,7 +20,7 @@ import SideImagePage from 'components/SideImagePage';
 import Page from 'components/Page';
 import { FixedFooterPage } from 'components/Page';
 import ContextProvider from 'components/ContextProvider';
-import Error from 'components/Error';
+import Error from './components/Error';
```
[the remark this answers is not in the document]
Comment (evans-g-crsj) [reply src/App.js:23]: Good point. I did not notice. Thanks Intellij

File: src/components/Login/roleForm.css (hunk 1 above)
Comment (evans-g-crsj) [reply src/components/Login/roleForm.css:6]: It's specific to one form but it's a good idea. I'll post-pone when the ticket for the theme will be done.

**Pull request merged**